ts/features/bonus/cgn/components/CgnCard.tsx

```diff
@@ -61,15 +61,19 @@ export const CgnCard = ({ expireDate, withEycaLogo }: CgnCardProps) => {
           <H6>{I18n.t("bonus.cgn.name")}</H6>
           {isExpired && expiredTag}
         </View>
-        <LabelSmall color="blueItalia-850" style={{ width: "70%" }}>
+        <BodySmall
+          weight="Semibold"
+          color="blueItalia-850"
+          style={{ width: "70%" }}
+        >
           {I18n.t("bonus.cgn.departmentName")}
-        </LabelSmall>
-        <LabelSmall color="blueItalia-850">
+        </BodySmall>
+        <BodySmall weight="Semibold" color="blueItalia-850">
```

Comment:
Why here you've add `weight="Semibold"`? 🤔

Reply:
@Ladirico The default weight of `LabelSmall` was `Semibold` too, so we apply the same logic applied to the `Body` style